tools/python-mock-server/responses/https.kubernetes.default.svc.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
is there a reason to rename it?

Reply:
One for test https route mocked response and one for the http mocked response